src/Core/src/Platform/Windows/HandlerExtensions.cs

```diff
@@ -37,7 +37,7 @@ public static FrameworkElement ToNative(this IElement view, IMauiContext context
 			var handler = view.Handler;
 
 			if (handler == null)
-				handler = context.Handlers.GetHandler(view.GetType());
+				handler = context.Handlers.GetHandler(TrimmerHelper.GetType(view));
```

Comment:
There are possible solutions to this which would NOT require a blind suppression of all warnings.
We should discuss this - and if there's no solution, it might be worth looking into possibly providing a new one.

Reply:
Sounds good, I'll be back from FTO on Monday. Thanks for the review!